### DISCUSSION TITLE
Welcome to vuejs.org Discussions!

### DISCUSSION THREAD

**Discussion by MachinisteWeb** [vuejs-fr/vuejs.org#236] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mdartic** [comment]
Hi @Haeresis 
Thanks to always be there !
Do you prefer we write in English or in French, as it's the fr translation ?
I'm here because I helped some times ago to translate the doc°.
Not really active in these times.

Do you know if we need to start translating the vue3 doc° ?
